limits/aio/storage/redis.py

```diff
@@ -148,7 +152,12 @@ def __init__(self, uri: str, **options) -> None:
         super().__init__()
 
         self.dependency = self.dependencies["coredis"]
-        self.storage = self.dependency.StrictRedis.from_url(uri, **options)
+        if connection_pool:
+            self.storage = self.dependency.StrictRedis(
+                connection_pool=connection_pool, **options
```

Comment:
https://github.com/redis/redis-py/blob/master/redis/client.py#L1211

strictly defining "StrictRedis" is pointless.... as it's just an alias to Redis

Reply:
Ah good point, I'd forgotten I'd pinned `redis-py` to `>3`

Reply:
actually no, sorry - you confused me as well :) That is the asyncio storage.

Reply:
aah sorry lol, i  was reading the code, didnt see teh aio in there